### ISSUE TITLE
Allow MySQL user length upto 32 characters

### ISSUE THREAD

**Issue by ram108** [serghey-rodin/vesta#1621]
The maximum length for a user name in MySQL does depend on the version, as stated in the manual:
http://dev.mysql.com/doc/refman/5.7/en/user-names.html

MySQL user names can be up to 32 characters long (16 characters before MySQL 5.7.8).

Please consider to extend default user length in VestaCP from 16 to 32 characters.

Request reference: https://github.com/serghey-rodin/vesta/issues/1023.

**Comment by dpeca** [comment]
I will take a care of it soon.

**Comment by mehargags** [comment]
Yes... this is much needed revision as all major Distributions (CentOS 7, Debian 9, Ubuntu 16.07) use MariaDB by default which supports even upto 80 chars. and now even official MySQL 5.7 supports upto 32.

I think making it upto 30 would be a safe bet.

**Comment by desfire** [comment]
Would also be great to make MariaDB 10.2 by default.. can't believe it is still MariaDB 5

**Comment by ram108** [comment]
Package installation depends by OS not Vesta.

**Comment by Luc45** [comment]
Just bumping this to say that I would like this too. Thanks.

**Comment by lukapaunovic** [comment]
@dpeca BUMP

**Comment by PecceG2** [comment]
Death? :-1:

**Comment by skullwritter** [comment]
dunno

**Comment by skullwritter** [comment]
@lukapaunovic i believe so, but neither i have lately time to commit here... also we have that problem that you referred.

**Comment by mtacore** [comment]
Honestly, you should give HestiaCP a try (VestaCP fork). It uses MariaDB 10.3 by default and has the length increased (among other fixes and enhancements). Just look for it at GitHub.

**Comment by anton-reutov** [comment]
MySQL User 	 32 version from 5.7.8
MariaDB User 32 version from 5.5.31
